docs/farming.md

```diff
@@ -188,6 +188,80 @@ After this, simply repeat the step you prompted for (step 4 or 6). This time, cl
 ```
 7. It may prompt again in here. Refer to the note on step 4.
 
+## Docker Instructions
+
+Create `subspace` directory and `docker-compose.yml` in it with following contents:
+```yml
+version: "3.7"
+services:
+  node:
+    # Replace `snapshot-DATE` with latest release
+    image: ghcr.io/subspace/node:snapshot-DATE
+    volumes:
+# Instead of specifying volume (which will store data in `/var/lib/docker`), you can
+# alternatively specify path to the directory where files will be stored, just make
+# sure everyone is allowed to write there
+      - node-data:/var/subspace:rw
+#      - /path/to/subspace-node:/var/subspace:rw
+    ports:
+# If port 30333 is already occupied by another Substrate-based node, replace all
+# occurrences of `30333` in this file with another value
+      - "0.0.0.0:30333:30333"
+# Un-comment following line to unlock node's WebSocket RPC
+#      - "127.0.0.1:9944:9944"
+    restart: unless-stopped
+    command: [
+      "--chain", "testnet",
+      "--base-path", "/var/subspace",
+      "--wasm-execution", "compiled",
+      "--execution", "wasm",
+      "--bootnodes", "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr",
+      "--port", "30333",
+      "--telemetry-url", "wss://telemetry.polkadot.io/submit/ 1",
+      "--telemetry-url", "wss://telemetry.subspace.network/submit/ 1",
+      "--rpc-cors", "all",
+      "--rpc-methods", "unsafe",
+      "--ws-external",
+      "--validator",
+# Replace `INSERT_YOUR_ID` with your node ID (shown in telemetry)
+      "--name", "INSERT_YOUR_ID"
+    ]
+
+  farmer:
+# Replace `snapshot-DATE` with latest release
+    image: ghcr.io/subspace/farmer:snapshot-DATE
+# Un-comment following 2 lines to unlock farmer's RPC
+#    ports:
+#      - "127.0.0.1:9955:9955"
+# Instead of specifying volume (which will store data in `/var/lib/docker`), you can
+# alternatively specify path to the directory where files will be stored, just make
+# sure everyone is allowed to write there
+    volumes:
+      - farmer-data:/var/subspace:rw
+#      - /path/to/subspace-farmer:/var/subspace:rw
+    restart: unless-stopped
+    command: [
+      "farm",
+      "--node-rpc-url", "ws://node:9944",
+      "--ws-server-listen-addr", "0.0.0.0:9955",
+# Replace `WALLET_ADDRESS` with your Polkadot.js wallet address
+      "--reward-address", "WALLET_ADDRESS"
+    ]
+volumes:
+  node-data:
+  farmer-data:
+```
+
+Now edit created file:
+1. Replace `snapshot-DATE` with the latest release (not pre-release!) snapshot
```

Comment:
I suggest we add an example here:
like: `ghcr.io/subspace/farmer:2022-mar-09` to avoid confusion and potential questions on **what should be the format of the date?** and such

Reply:
Also, the same example should be placed in line 198 imo